### DISCUSSION TITLE
Provide Instructions for using .sha256sum.txt and .sig files

### DISCUSSION THREAD

**Discussion by MSIH** [microsoft/openjdk#303] [Q&A] [answered]
With the importance of checking the integrity of install files from vendors at an all-time high, I think would be helpful to the community to provide detailed guidance on how to use the .sha256sum.txt and .sig files to verify the integrity of the install package.

Some would argue that it makes the most sense to have an individual page with this information that download pages can link to. 

I would disagree, the point is to get people to check the integrity, so I suggest the directions be on the page itself. 

Windows : Get-FileHash -Algorithm SHA256 -Path 'microsoft-jdk-11.0.16-windows-x64.msi'
iOS: ?
Linux/Unix: ?

**Comment by karianna** [comment]
CC @brunoborges - for your consideration

**Comment by KalleOlaviNiemitalo** [comment, marked as answer]
<https://aka.ms/download-jdk/microsoft-jdk-17.0.4-linux-aarch64.tar.gz.sha256sum.txt> contains

```
c661f20f883c5b69c680736d71309d00551ad988f8f7489a1cb0b6abc4291314  microsoft-jdk-17.0.4-linux-aarch64.tar.gz
```

which looks like the format understood by `sha256sum -c` (see [sha256sum(1) - Linux manual page](https://man7.org/linux/man-pages/man1/sha256sum.1.html)) or `shasum -a 256 -c` (see [shasum - Print or Check SHA Checksums - Perldoc Browser](https://perldoc.perl.org/shasum)).
